rise/static/main.js

```diff
@@ -615,25 +671,36 @@ function revealMode() {
   }
 }
 
+// users can bind another key with e.g.
+// Jupyter.notebook.keyboard_manager.command_shortcuts.set_shortcut('alt-a', 'RISE:slideshow');
+// and likewise bind helper actions with e.g.
+// Jupyter.notebook.keyboard_manager.command_shortcuts.set_shortcut('shift-i', 'RISE:toggle-slide');
+    
```

Comment:
Maybe we should set the shortcut by default? There were some request about that in the past: #114

Reply:
well this is really up to you, as I mentioned in the past, chosing the right keyboard shortcut is kind of tricky, and personal too I believe, and I'd rather not waste time discussing these. 
If you are fine with the ones in the comments we can use this of course, people will easily be able to change them later on (was the whole point for this PR in the first place ;)

Reply:
Yes, I think we need to add default values and I am happy with the ones you have in the comments... and as you said, people can change them if they disagree with the choice.

Reply:
FYI, I added default values for slide, subslides and fragments in another branch.